conans/client/output.py

```diff
@@ -8,12 +8,29 @@
 
 
 def colorama_initialize():
+    if "NO_COLOR" in os.environ:
+        return False
+
+    clicolor_force = get_env("CLICOLOR_FORCE")
+    if clicolor_force is not None and clicolor_force != "0":
```

Comment:
Thy not using just `get_env()` here https://docs.conan.io/en/latest/reference/tools.html#tools-get-env
```suggestion
    if get_env("CLICOLOR_FORCE"):
```